config.json

```diff
@@ -53,5 +53,9 @@
     "dataDaemon": {
         "bindAddress": "localhost",
         "port": 9991
+    },
+    "recordLog": {
+        "enabled": false,
+        "recordLogName": "main"
```

Comment:
Since this is the default and a lot of people don't change defaults, it's better to give it a specific name. May be
`s3-recordLog`?